rclpy/test/test_node.py

```diff
@@ -681,7 +700,7 @@ def test_node_get_parameters_by_prefix(self):
 
         parameters = self.node.get_parameters_by_prefix('')
         self.assertIsInstance(parameters, dict)
-        self.assertEqual(len(parameters), 6)
```

Comment:
@jubeira why was this assertion removed?

Reply:
It's actually redundant since I'm comparing the complete dictionary below, so it's not adding any value.